src/lib/plot/series/bar-series.js

```diff
@@ -83,33 +83,36 @@ class BarSeries extends AbstractSeries {
   _updateSeries() {
     const container = getDOMNode(this.refs.container);
     const {
-      sameTypeTotal = 1,
-      sameTypeIndex = 0,
-      data} = this.props;
-
-    const {
+      _stackBy,
+      data,
       lineSizeAttr,
       valuePosAttr,
       linePosAttr,
       valueSizeAttr} = this.props;
 
+    let {
+      sameTypeTotal = 1,
+      sameTypeIndex = 0} = this.props;
+
     if (!data || !data.length) {
       return;
     }
 
     const distance = this._getScaleDistance(linePosAttr);
-    const baseValue = this._getScaleBaseValue(valuePosAttr);
     const lineFunctor = this._getAttributeFunctor(linePosAttr);
     const valueFunctor = this._getAttributeFunctor(valuePosAttr);
+    const value0Functor = this._getAttr0Functor(valuePosAttr);
+
+    if (_stackBy === valuePosAttr) {
+      sameTypeTotal = 1;
+      sameTypeIndex = 0;
```

Comment:
Hmm.. how come this line is needed? Does this mean that these are reset whenever stacking by the value attribute? Perhaps a comment?